---
layout: page
title: "Resources"
#subheadline: ""
#teaser: ""
#header:
#   image_fullwidth: "header_roadmap_3.jpg"
permalink: "/resources/"
comments: false
show_meta: false
---

#### <i class="fa fa-search"></i> [Looking for available positions ?]({{ site.url }}/positions) <i class="fa fa-users"></i>

## Tools:

### [PRADA](http://bioinformatics.mdanderson.org/main/PRADA:Overview)

Massively parallel sequencing of cDNA reverse transcribed from RNA (RNASeq) provides an accurate estimate of the quantity and composition of mRNAs. To characterize the transcriptome through the analysis of RNA-seq data, we developed PRADA. PRADA focuses on the processing and analysis of gene expression estimates, supervised and unsupervised gene fusion identification, and supervised intragenic deletion identification. The BAM files generated by the pipeline are readily compatible with different tools for mutation calling and to obtain read counts for further downstream analysis.

>Publication: Torres-Garcia W, Zheng S, .., Verhaak RG. PRADA: pipeline for RNA sequencing data analysis. *Bioinformatics* 2014. PMID: [24695405](http://www.ncbi.nlm.nih.gov/pubmed/24695405)

## Database:

### [TumorFusions](http://tumorfusions.org)

Transcripts fusion as a result of genomic rearrangement is an important class of somatic alteration, as a cancer initiating event and as a molecular therapeutic target for specific tumors. Our Pipeline for RNA sequencing Data Analysis (PRADA) enables us to detect fusion transcripts with high confidence comprehensively. Based on integrated analysis of paired-end RNA sequencing and DNA copy number data from The Cancer Genome Atlas(TCGA), TCGA Fusion Gene Data Portal provides a bona-fide fusion list across many tumor types.

>Publication: Yoshihara, K. Wang Q, .., Verhaak RG. The landscape and therapeutic relevance of cancer-associated transcript fusions. Oncogene. 2014. PMID: [25500544](http://www.ncbi.nlm.nih.gov/pubmed/25500544)



